### ISSUE TITLE
Leder på Naturkartan

### ISSUE THREAD

**Issue by dpriskorn** [dpriskorn/svenska-vandringsleder#72]
alla (1566 i skrivande stund) har skrapats se https://github.com/dpriskorn/NaturkartanScraper/blob/master/trails.csv

**Comment by dpriskorn** [comment]
runt 800 leder från naturkartan har nu importerats.

**Comment by salgo60** [comment]
Funderar på att skriva till massa inblandade för en känsla att massa utbetalningar sker från Tillväxtverket till en del av Business Stockholm som heter [Stockholm Archipelago](https://github.com/salgo60/Stockholm_Archipelago_Trail/issues?q=state%3Aopen%20label%3A%22Stockholm%20Archipelago%20-%20samverkansorganisation%22)  och leveransen ärmenormt dålig... verkar som Tillväxtverket fyller i vad som levererats se [#222](https://github.com/salgo60/Stockholm_Archipelago_Trail/issues/146#issuecomment-3354830020)
* vore bra på att visa hur mycket data som finns som data hos OSM...
* var hittar jag dessa leder du ladda upp ? ...

**Comment by dpriskorn** [comment]
> Funderar på att skriva till massa inblandade för en känsla att massa utbetalningar sker från Tillväxtverket till en del av Business Stockholm som heter [Stockholm Archipelago](https://github.com/salgo60/Stockholm_Archipelago_Trail/issues?q=state%3Aopen%20label%3A%22Stockholm%20Archipelago%20-%20samverkansorganisation%22)  och leveransen ärmenormt dålig... verkar som Tillväxtverket fyller i vad som levererats se [#222](https://github.com/salgo60/Stockholm_Archipelago_Trail/issues/146#issuecomment-3354830020)
> * vore bra på att visa hur mycket data som finns som data hos OSM...

Cirka hälften av alla lederna inkl etapper finns i OSM i dagsläget. Matchern kördes senast i förrgår. 😀

> * var hittar jag dessa leder du ladda upp ? ...

https://www.wikidata.org/wiki/Wikidata:WikiProject_Hiking_trails/Sweden/Coverage
Där kan du filtrera på både Naturkartan och OSM.

Huvudorsaken till att etapper idag saknas i OSM är att det saknas GPX med rätt licens från kommuner och regioner. Vi får inte rita av efter Naturkartan eftersom de hävdar att datan är upphovsrättsskyddat (de har Copyright på alla sidor) vilket är mycket märkligt.

Regeringen skulle kunna sätta ner foten och göra det olagligt att dela offentliga data med Copyright via ett privat företag när PRV rekommenderar CC0, men i dagsläget är det knäpptyst uppifrån. Naturkartan som företag är inte problemet utan det är troligtvis brist på kompetens vid upphandling och dålig kravställning vid avtals förhandlingarna som lett till situationen vi befinner oss i idag.
Alla organisationerna som skrivit avtal hittills verkar ha missat att ta upp licens frågan. 🤷‍♂️

Jag har importerat ca 1000 leder nu. För varje led dom finns i Naturkartan men saknas i OSM skulle vi kunna begära ut data från berörda kommuner/regioner om vi orkar.

Jag undrar hur mycket pengar vi går miste om i uteblivna intäkter i turismsektorn pga dålig data om våra leder som underhålls med skattemedel.
Ska vi be Chatgpt räkna på det?

Vi skulle kunna begära ut avtalet mellan tex region Blekinge och Naturkartan och se om det står något om licens. De fick massor med lona medel om jag minns rätt och vägrade dela när jag ringde och slängde på luren. Min intuition säger mig att om en offentlig anställd slänger på luren i ilska, då ligger det en hund begravet i närheten...

**Comment by salgo60** [comment]
@dpriskorn  Jag var med på [OSM återkommande community möte](https://community.openstreetmap.org/t/svenskt-aterkommande-communitymote/113856/36) och då var en kommun med från Blekinge kommun och dom hade försökt få ut sitt eget data från Naturkartan och blivit nekade...

se 
* [Vilken kommun är det som inte fick ut sitt data från naturkartan?](https://community.openstreetmap.org/t/vilken-kommun-ar-det-som-inte-fick-ut-sitt-data-fran-naturkartan/130377)

<img width="819" height="302" alt="Image" src="https://github.com/user-attachments/assets/e9b312b1-974c-4e18-b8e6-ce934fc80ca8" />

* Lucas verkar ha hoppat vidare [Linked in](https://www.linkedin.com/in/lucas-liebler-9537081bb)

<img width="791" height="217" alt="Image" src="https://github.com/user-attachments/assets/48a89bdd-90bb-4479-9174-d367790d8bea" />

**Comment by salgo60** [comment]
@dpriskorn kan det finnas spår med rätt licens på Strava eller annat?

**Comment by dpriskorn** [comment]
> [@dpriskorn](https://github.com/dpriskorn) kan det finnas spår med rätt licens på Strava eller annat?

nej, Strava spår är inte öppet data. Man kan gissa var leder finns genom att titta på dem och sen gå ut och inventera, men får inte rita direkt från strava.
